applications/StructuralMechanicsApplication/custom_elements/small_displacement.h

```diff
@@ -232,18 +233,6 @@ class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION) SmallDisplacement
     ///@name Private Operators
     ///@{
 
-    void CalculateBodyForces(
```

Comment:
You don't need these three functions any more? From what I can see they are not in the baseclass

Reply:
Nope, are defined but not used, so I removed in order to clean a little bit

Reply:
👍